osbrain/proxy.py

```diff
@@ -100,6 +100,30 @@ def _ready_or_timeout(self, time0, timeout):
             raise
         return True
 
+    def wait_for_running(self, timeout=3.):
+        """
+        Wait until the agent is running.
+
+        Parameters
+        ----------
+        timeout : float
+            If not negative, raise an error if the agent is not running
+            after that number of seconds.
```

Comment:
Maybe: "Raise an exception if the agent is not running after this number of seconds. Use a negative value to wait indefinitely." (o forever). What you think?

Reply:
Sounds better. I'll change it